src/pages/SetPasswordPage.js

```diff
@@ -82,7 +82,7 @@ class SetPasswordPage extends Component {
     render() {
         return (
             <SafeAreaView style={[styles.signInPage]}>
-                <SignInPageLayout>
+                <SignInPageLayout welcomeText={this.props.translate('welcomeText.phrase4')}>
```

Comment:
You'll have to change this when you update the translations as per my other comment.

Reply:
Updated translate ref.